src/main/60-eks.tf

```diff
@@ -66,6 +66,12 @@ module "eks" {
         computeType = "Fargate"
       })
     }
+    adot = {
+      addon_version               = var.kubernetes_addons_versions.adot
+      resolve_conflicts_on_create = "OVERWRITE"
+      resolve_conflicts_on_update = "PRESERVE"
+
+    }
```

Comment:
Are we using the addon? It looks like we are creating ADOT with yaml manifests in `k8s/assets/adot_collector`

Reply:
This is the operator installation. In the k8s section is the collector installation

Reply:
Do you need both? We usually scrape cadvisor metrics and kube state metrics using only `aws-otel-collector`. If you feel that both are needed in this case that's okay, as long as metrics are not duplicated